pupil_src/shared_modules/head_pose_tracker/controller/online_controller.py

```diff
@@ -60,6 +62,17 @@ def _calculate_current_pose(self, frame):
             self._camera_intrinsics,
         )
 
+    def _create_position_events(self, current_pose):
+        """
+        Adds position events to the current list of events.
```

Comment:
Please adjust the description, as the function does not add anything to `events`.